src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/integration/wiremock/PrisonApiMockServer.kt

```diff
@@ -17,7 +21,19 @@ import uk.gov.justice.digital.hmpps.hmppsmanageadjudicationsapi.integration.IntT
 import uk.gov.justice.digital.hmpps.hmppsmanageadjudicationsapi.integration.IntTestData.Companion.UPDATED_LOCATION_ID
 import uk.gov.justice.digital.hmpps.hmppsmanageadjudicationsapi.integration.IntTestData.Companion.UPDATED_STATEMENT
 
-class PrisonApiMockServer : WireMockServer(8979) {
+class PrisonApiMockServer : WireMockServer {
+  constructor() : super(8979) {
```

Comment:
This logging was really useful in detecting errors in mocking data, so I have kept it in.